internal/resource/url.go

```diff
@@ -321,8 +322,10 @@ func (f *Fetcher) FetchFromS3(u url.URL, dest *os.File, opts FetchOptions) error
 
 	if f.AWSSession == nil {
 		var err error
+		maxRetries := int(10)
 		f.AWSSession, err = session.NewSession(&aws.Config{
 			Credentials: credentials.AnonymousCredentials,
+			MaxRetries:  &maxRetries,
```

Comment:
This is slightly more readable as `aws.Int(10)` if we're hardcoding it anyways.

For reference, this is bumping from the default of 3.

With or without my slight readability nit, this LGTM.

Reply:
I am curios how the number 10 was picked though; if you have any information leading to that being a better number than 3 or 5, I'd be interested in hearing it.

I assume the SDK's value was chosen for a reason (especially given all the sdks share 3 for s3, use 10 for certain dynamodb requests, etc etc)

Reply:
Bad answer: it's an arbitrary number bumping over the initial default (both for S3 and TFTP). Actual number was a hint from the initial ticket, but that was arbitrary as well. Both clients already do retries so this PR just ups, uniforms and makes the number explicit.